confgenerator/config.go

```diff
@@ -574,6 +578,10 @@ type OTelReceiver interface {
 	Pipelines(ctx context.Context) []otel.ReceiverPipeline
 }
 
+type ModifiableOtelReceiver interface {
+	MergeWithProcessor(ctx context.Context, processor MetricsProcessor) []otel.ReceiverPipeline
```

Comment:
If an interface has one method it should be named after the method (maybe `MetricsProcessorMerger` in this case)

Reply:
Done!